grabbit/src/main/groovy/com/twcable/grabbit/client/batch/ClientBatchJobExecutionListener.groovy

```diff
@@ -120,7 +121,8 @@ class ClientBatchJobExecutionListener implements JobExecutionListener{
 
         final String encodedContentAfterDate = URLEncoder.encode(contentAfterDate, 'utf-8')
         final String encodedPath = URLEncoder.encode(path, 'utf-8')
-        final grabPath = "http://${host}:${port}/grabbit/job?path=${encodedPath}&after=${encodedContentAfterDate}"
+        final String encodedExcludedPaths = URLEncoder.encode(excludePaths, 'utf-8')
+        final grabPath = "http://${host}:${port}/grabbit/job?path=${encodedPath}&excludePaths=${encodedExcludedPaths}&after=${encodedContentAfterDate}"
```

Comment:
FWIW, on the specific example of what if `port` is blank, maybe a better solution to that is to explicitly map a blank port value to `80`.